ArdusatSDK.cpp

```diff
@@ -1,17 +1,35 @@
-/**
- * @file   ArdusatSDK.cpp
- * @Author Ben Peters (ben@ardusat.com)
- * @date   December 3, 2014
- * @brief  Implements ArdusatSDK generic sensor read & configuration functions
+/*
+ * =====================================================================================
+ *
+ *       Filename:  ArdusatSDK.cpp
+ *
+ *    Description:  Implements ArdusatSDK generic sensor read and configuration
+ *                  functions for Space Kit Sensors.
+ *
+ *                  This SDK uses many third-party libraries available from
+ *                  Adafruit (https://github.com/adafruit). These libraries are
+ *                  mostly under an Apache License, Version 2.0.
+ *
+ *                  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Organization:  Ardusat
+ *        Version:  2.0
+ *       Compiler:  Arduino
+ *
+ *        Created:  12/3/2014
+ *         Author:  Ben Peters (ben@ardusat.com)
+ *   Last Updated:  8/25/2015
+ *   Last Updater:  Sam Olds (sam@ardusat.com)
+ *
+ * =====================================================================================
```

Comment:
Do you want to use Doxygen compatible comments for the example sketches as well? I assumed it would be more common to read the actual sketch files rather than reading some Doxygen docs about them.